src/MSAL.PCL/Internal/Cache/TokenCacheAccessor.cs

```diff
@@ -77,17 +77,17 @@ public IList<RefreshTokenCacheItem> GetRefreshTokens(TokenCacheKey tokenCacheKey
             return matchedRefreshTokens;
         }
 
-        public void DeleteToken(AccessTokenCacheItem accessToken‪Item)
+        public void DeleteAccessToken(AccessTokenCacheItem accessToken‪Item)
         {
-            TokenCachePlugin.DeleteToken(accessToken‪Item.GetTokenCacheKey());
+            TokenCachePlugin.DeleteAccessToken(accessToken‪Item.GetTokenCacheKey());
         }
 
         public void DeleteRefreshToken(RefreshTokenCacheItem refreshToken‪Item)
         {
             TokenCachePlugin.DeleteRefreshToken(refreshToken‪Item.GetTokenCacheKey());
         }
 
-        public IList<AccessTokenCacheItem> GetAllAccessTokens()
+        public IList<AccessTokenCacheItem> GetAllAccessTokens(string clientId)
         {
             ICollection<string> allTokensAsString = TokenCachePlugin.GetAllAccessTokens();
```

Comment:
if I pass raw strings then it would pose a challenge around how token cache entry is created in iOS where the key does not follow standard norm and dimensions of the key are written as keychain entry metadata. I would need to deserialize it again for iOS and peel out information which is not ideal. But as a principle, your suggestion is excellent. I am just bound by iOS mechanics